### ISSUE TITLE
Commits

### ISSUE THREAD

**Issue by matthewdking** [fac-13/PLHH-ToDo#56]
As much as I love this commit message...
![commit](https://user-images.githubusercontent.com/25408167/37203460-9cfb3c20-2385-11e8-9003-4bf6a43b23bb.gif)

Commit messages should be clear and descriptive of what you are doing as well as referencing an issue. The rest of your git flow looks great :ok_hand:

**Comment by haydnba** [comment]
Fair Point - to prove your point I can't really even understand what I'm referring to myself...
😭
